src/01-general/user-invites.apib

```diff
@@ -32,3 +32,14 @@ Cancel a user invite.
         + id: `e88131bf-50c4-49d3-8ae3-47e5d9626bf6` (string, required)
 
 + Response 204
+
+### userInvites.resend [GET /userInvites.resend]
+
+Resend a user invite.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + id: `e88131bf-50c4-49d3-8ae3-47e5d9626bf6` (string, required)
+
++ Response 204
```

Comment:
If this creates a new invitation this should be 201 with the new id in the response.